FrontEnd/SignUpPage/src/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -0,0 +1,267 @@
+                            <label className={styles["signup-form__label--required"]}>*</label>
+                            <div className={styles["signup-form__label--password"]}>
+                                <label>Пароль</label>
+                                <label className={styles["signup-form__label--hint"]}>(Повинен містити A-Z, a-z,
+                                    0-9)</label>
+                            </div>
+                        </div>
+                        <div className={styles["signup-form__field"]}>
+                            <input
+                                className={styles["signup-form__input"]}
+                                placeholder='Пароль'
+                                type="password"
+                                {...register('password', {
+                                    required: errorMessageTemplates['required'],
+                                    pattern: {
+                                        value: passwordPattern,
+                                        message: errorMessageTemplates['password']
+                                    }
+                                })}
+
+                            />
+                        </div>
+                        <div className={styles["signup-form__error"]}>
+                            {errors.password && errors.password.message}
+                        </div>
+                    </div>
+                    {/* Confirm password */}
+                    <div className={styles["signup-form__column"]}>
+                        <div className={styles["signup-form__label"]}>
+                            <label className={styles["signup-form__label--required"]}>*</label>
+                            <div className={styles["signup-form__label--password"]}>
+                                <label className={styles["signup-form__label--text"]}>Повторіть
+                                    пароль</label>
+                                <label className={styles["signup-form__label--hint"]}>(Повинен містити A-Z, a-z,
+                                    0-9)</label>
+                            </div>
+                        </div>
+                        <div className={styles["signup-form__field"]}>
+                            <input
+                                className={styles["signup-form__input"]}
+                                placeholder='Пароль'
+                                type="password"
+                                {...register('confirmPassword', {
+                                    required: errorMessageTemplates['required'],
+                                    validate: (value) => {
+                                        if (watch('password') !== value) {
+                                            return errorMessageTemplates['confirmPassword']
+                                        }
+                                    }
```

Comment:
could be simplified to
```suggestion
                                    validate: (value) => watch('password') !== value && errorMessageTemplates['confirmPassword'] 
```